scripts/sentiment_analysis/sentiment_analysis_cnn.py

```diff
@@ -0,0 +1,195 @@
+"""
+Fine-tune Language Model for Sentiment Analysis
+===============================================
+This example shows how to use convolutional neural networks (textCNN)
+for sentiment analysis on various datasets.
+"""
+
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import argparse
+import time
+import random
+
+import mxnet as mx
+from mxnet import nd, gluon, autograd
+from mxnet.gluon.data import DataLoader
+import numpy as np
+import process
+import textCNN
```

Comment:
import .process
import .textCNN

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

You should put a blank line between each group of imports. Sort imports alphabetically within each group.